wiki/Tournaments/CES/CES2/en.md

```diff
@@ -0,0 +1,119 @@
+---
+tags:
+  - CES
+---
+
+# CES League osu! Tournament 2
+
+![CES League osu! Tournament 2 Logo](img/logo.jpg)
+
+The **CES League osu! Tournament 2** (***CES2***) is an international 1v1 double-elimination osu! tournament hosted by ![][flag_NL] [Stan](https://osu.ppy.sh/users/11212255). It features one of the largest staff team in the history of osu! tournaments. This is the second installment of the CES League osu! Tournament series.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2020-05-01/2020-05-24 |
+| Qualifiers | 2020-06-05/2020-06-07 |
+| Round of 128 | 2020-06-12/2020-06-21 |
+| Round of 64 | 2020-06-26/2020-07-05 |
+| Round of 32 | 2020-07-10/2020-07-12 |
+| Round of 16 | 2020-07-17/2020-07-19 |
+| Quarterfinals | 2020-07-24/2020-07-26 |
+| Semifinals | 2020-07-31/2020-08-02 |
+| Finals | 2020-08-07/2020-08-09 |
+| Grand Finals | 2020-08-14/2020-08-16 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Profile Badge (Pending) + Profile Banner + $130.00 (or 5 years of osu! supporter) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | Profile Banner + $60.00 (or 2 years of osu! supporter) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | Profile Banner + $30.00 (or 1 year of osu! supporter) |
+
+![](img/badge.png "CES2 winner badge")
+
+## Organisation
+
+The CES League osu! Tournament 2 was run by various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Admin | ![][flag_NL] [Stan](https://osu.ppy.sh/users/11212255) |
```

Comment:
```suggestion
| Administrator | ![][flag_NL] [Stan](https://osu.ppy.sh/users/11212255) |
```
Either Administrator or you can do Organiser (I usually prefer this) or Manager too.